mklive.sh.in

```diff
@@ -75,6 +75,8 @@ directory if unset).
  -l <locale>        Default locale to use (en_US.UTF-8 if unset).
  -i <lz4|gzip|bzip2|xz> Compression type for the initramfs image (xz if unset).
  -s <gzip|lzo|xz>     Compression type for the squashfs image (xz if unset)
+ -S <shell>         Default shell of the root user (must be absolute path). 
+                    Set the live.shell kernel parameter to change the default shell of anon.
```

Comment:
Since this PR was made, https://github.com/void-linux/void-mklive/pull/267 was merged and added a -S argument.

Reply:
I changed the flag to `-e`. The flag letters are running out.